app/services/food_db_service.rb

```diff
@@ -0,0 +1,19 @@
+class FoodDBService
+  def self.get_searched_foods(search_query)
```

Comment:
No need to have 2 separate methods for the different purposes (searched food vs total hits) as they are the same endpoint.